tango_simlib/tests/test_fandango_json_parser.py

```diff
@@ -37,52 +37,32 @@
 # The desired information for the attribute 'Status' when the database2
 # json file is parsed by the FandangoExportDeviceParser.
 EXPECTED_STATUS_ATTR_INFO = {
-    "alarms": {
-        "delta_t": "Not specified",
-        "delta_val": "Not specified",
-        "extensions": "[]",
-        "max_alarm": "Not specified",
-        "max_warning": "Not specified",
-        "min_alarm": "Not specified",
-        "min_warning": "Not specified",
-    },
-    "color": "Lime",
+    "abs_change": "Not specified",
+    "archive_abs_change": "Not specified",
+    "archive_period": "Not specified",
+    "archive_rel_change": "Not specified",
     "data_format": tango._tango.AttrDataFormat.SCALAR,
     "data_type": tango._tango.CmdArgType.DevString,
-    "database": "monctl:10000",
+    "delta_t": "Not specified",
+    "delta_val": "Not specified",
     "description": "",
-    "device": "sys/database/2",
     "display_unit": "No display unit",
     "enum_labels": [],
-    "events": {
-        "arch_event": {
-            "archive_abs_change": "Not specified",
-            "archive_period": "Not specified",
-            "archive_rel_change": "Not specified",
-            "extensions": "[]",
-        },
-        "ch_event": {
-            "abs_change": "Not specified",
-            "extensions": "[]",
-            "rel_change": "Not specified",
-        },
-        "per_event": {"extensions": "[]", "period": "1000"},
-    },
-    "format": "%s",
```

Comment:
Should we remove `format`?  I see it in the attribute config that we can read back. 

```python
dp.get_attribute_config_ex("desiredPointing")
Out[5]: [AttributeInfoEx(alarms = AttributeAlarmInfo(delta_t = 'Not specified', delta_val = 'Not specified', extensions = [], max_alarm = 'Not specified', max_warning = 'Not specified', min_alarm = 'Not specified', min_warning = 'Not specified'), data_format = tango._tango.AttrDataFormat.SPECTRUM, data_type = tango._tango.CmdArgType.DevDouble, description = 'No description', disp_level = tango._tango.DispLevel.OPERATOR, display_unit = 'No display unit', enum_labels = [], events = AttributeEventInfo(arch_event = ArchiveEventInfo(archive_abs_change = 'Not specified', archive_period = 'Not specified', archive_rel_change = 'Not specified', extensions = []), ch_event = ChangeEventInfo(abs_change = 'Not specified', extensions = [], rel_change = 'Not specified'), per_event = PeriodicEventInfo(extensions = [], period = '1000')), extensions = [], format = '%6.2f', label = 'desiredPointing', max_alarm = 'Not specified', max_dim_x = 3, max_dim_y = 0, max_value = 'Not specified', memorized = tango._tango.AttrMemorizedType.NONE, min_alarm = 'Not specified', min_value = 'Not specified', name = 'desiredPointing', root_attr_name = 'Not specified', standard_unit = 'No standard unit', sys_extensions = [], unit = '', writable = tango._tango.AttrWriteType.READ_WRITE, writable_attr_name = 'desiredpointing')]
```

Reply:
Yes. It is something that tango create automatically depending on the `data_type`. We do not need it to create an attribute dynamically.